### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by p-grzelczak**

### CODE REVIEW

**Comment by cheshireoctopus** [comment]
@p-grzelczak should we remove `package-lock.json` from this repo? Possible conflicts with `yarn.lock`?

**Review by cheshireoctopus — APPROVED** [review]
lg2m - curious though why there were so many changes to `lib` files here?

File: package.json [hunk 1]
```diff
@@ -31,23 +31,24 @@
   },
   "pre-commit": "lint:staged",
   "dependencies": {
-    "axios": "^0.19.0",
+    "@babel/cli": "^7.11.6",
```
Comment (adybionka) [inline package.json:34]: shouldn't be in the devDependencies?

**Comment by adybionka** [comment]
> lg2m - curious though why there were so many changes to `lib` files here?

A different version of babel/cli produces different transpiled JS code.

**Review by xiv — APPROVED** [review]
LGTM

**Comment by p-grzelczak** [comment]
Not doing this anymore, will soon merge api-client into monorepo

**Pull request closed without merging**